pkg/question/question.go

```diff
@@ -1160,26 +1222,73 @@ func AskInstanceIds(h *ec2helper.EC2Helper, addedInstanceIds []string) (*string,
 	return &answer, err
 }
 
+func AskBootScriptConfirmation(h *ec2helper.EC2Helper, defaultBootScript string) string {
+	stringOptions := []string{cli.ResponseYes, cli.ResponseNo}
+	optionsText := yesNoOption + "\n"
+	question := "Would you like to add a filepath to the instance boot script?"
+	defaultOption := aws.String(cli.ResponseNo)
+	if defaultBootScript != "" {
+		defaultOption = aws.String(cli.ResponseYes)
+	}
+
+	answer := AskQuestion(&AskQuestionInput{
+		QuestionString: question,
+		DefaultOption:  defaultOption,
+		EC2Helper:      h,
+		Fns:            []CheckInput{ec2helper.ValidateFilepath},
+		OptionsString:  &optionsText,
+		StringOptions:  stringOptions,
+	})
+	return answer
```

Comment:
I agree about the unnecessary `answer` variable. However, I'm not going to change them now because the upcoming PRs to add Bubble Tea support will rework these methods to have an additional `error` return value. At that point, it will be clearer to use a standalone `return` statement.